.packit.yml

```diff
@@ -0,0 +1,18 @@
+---
+specfile_path: nmstate.spec
+upstream_package_name: nmstate
+upstream_project_url: http://nmstate.io
+create_tarball_command: ["python3", "setup.py", "sdist", "--dist-dir", "."]
+current_version_command: ["python3", "setup.py", "--version"]
+create_pr: true
+actions:
+  post-upstream-clone: "bash -c './packaging/make_spec.sh > nmstate.spec'"
+jobs:
+  - job: copr_build
+    trigger: pull_request
+    metadata:
+      targets:
+        - centos-stream-x86_64
+        - epel-8-x86_64
+        - fedora-32-x86_64
```

Comment:
You can use `fedora-all`, `fedora-devel`, or `fedora-stable` aliases to not need to change anything on each release. ([Updated manually](https://github.com/packit-service/packit/pull/731) on our site for the last time..;)

Aliases will be used with the `x86_64` by default, but you can use something like `fedora-stable-aarch64` as well.

Reply:
We are not targeting Fedora 31 currently, therefore it seems to be the right choice to specify the chroot's explicitly. Actually I guess we do not need Fedora builds currently for the PRs, so I removed them for now.